Automattic/wp-calypso
# Domain transfer - Redirect logged out users to login

Related to #79286

## Proposed Changes

Redirect logged-out users to login. More [context](pdtkmj-1Cx-p2#comment-2928)
Logged-out users were getting  “invalid authentication code” even if the code was correct, since they need to be logged in to purchase the transfer, this PR checks and redirects to log in.

## Testing Instructions

- Pull and run this branch or user calypso live link
- Use incognito
- Navigate to `/setup/domain-transfer/domains`
- It should redirect you to login and then `setup/domain-transfer` to start the flow
